priv/perf/docker/k8s_job.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: perf-job
+spec:
+  template:
+    spec:
+      containers:
+      - name: perf
+        image: perf:lo
+        imagePullPolicy: Never
+        command: ["mix",  "run", "-e", "LoadTest.TestRunner.run()", "--", "deposits", "1", "5"]
+        env:
```

Comment:
I'm not sure how should I set them. should I hardcode them and create different files for different env. or should I use something like https://kubernetes.io/docs/concepts/configuration/configmap/

Reply:
Later in the stage it makes sense to port those env non-changing or quite likely wouldn't change values into config file so we can just have them in the cluster that runs the job. But I think starting with file per test is good for now.